### PR TITLE
Releases for 1Q 2017

### PR DESCRIPTION
**Pull request by adriaanm**
TODO: 
  - [X] add download page for 2.13.0-M1
  - [x] add 2.13.x nightlies links to documentation/api.md

### CODE REVIEW

**Comment by SethTisue** [comment]
TODO: add 2.13.x nightlies links

**Comment by adriaanm** [comment]
Updated. Should be ready to go tomorrow.

**Comment by ashawley** [comment]
> As part of our ongoing efforts at Lightbend to simplify contributing to Scala

Everything was moved to sbt, also.  That's a big deal, IMO. The sbt build for the compiler has been in use behind the scene for a couple years, but people may have not noticed the switch.  I believe this was the first official 2.11 release packaged using sbt, and the third with 2.12.

File: news/_posts/2017-04-18-releases-1Q17.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+We are happy to announce three new Scala releases: 2.11.11 concludes the 2.11 series, 2.12.2 brings bug fixes and small improvements, and 2.13.0-M1 sets the scene for modularization and improvements coming in the next major release next year. Many thanks to everyone who contributed!
+
+As part of our ongoing efforts at Lightbend to simplify contributing to Scala, we are thrilled to announce our move to GitHub for [bug reporting](https://github.com/scala/bug), as well as for [release notes](https://github.com/scala/scala/releases).
+
+For the highlights of these three releases, read on.
+
+<!-- break -->
+
+[Scala 2.11.11](https://github.com/scala/scala/releases/tag/v2.11.11) contains many smaller fixes and a few backports from 2.12. It also features improved error messages on missing dependencies; this improvement was developed by @jvican at the Scala Center under proposal SCP-009.
```
Comment (jvican) [inline news/_posts/2017-04-18-releases-1Q17.md]: Thank you for the mention!
  
Can we add at the end of this paragraph the following line: "The rest of implementation of SCP-009 was outsource to [a compiler plugin](https://github.com/scalacenter/classpath-shrinker)."?

**Comment by adriaanm** [comment]
> I believe this was the first official 2.11 release packaged using sbt

We're still using ant to build the release version of 2.11.

See e.g. https://scala-ci.typesafe.com/view/scala-2.11.x/job/scala-2.11.x-integrate-bootstrap/921/consoleFull:
```
### Bootstrapping Scala using locker
+++ cd /home/jenkins/workspace/scala-2.11.x-integrate-bootstrap
+++ rm -rf build/
+++ ant -Dstarr.version=2.11.11-bin-8a413ba -Dscala.full.version=2.11.11-bin-8a413ba -Dextra.repo.url=https://scala-ci.typesafe.com/artifactory/scala-integration/ -Dmaven.version.suffix=-bin-8a413ba -Dactors-migration.version.number=1.1.0 -Dscala-continuations-library.version.number=1.0.2 -Dscala-continuations-plugin.version.number=1.0.2 -Dscala-parser-combinators.version.number=1.0.4 -Dscala-swing.version.number=1.0.2 -Dscala-xml.version.number=1.0.5 -Dpartest.version.number=1.0.16 -Dscalacheck.version.number=1.11.6 -Dscala.binary.version=2.11 -Dupdate.versions=1 -Dscaladoc.git.commit=8a413ba7cc7a32210fc2d734c18032b718035116 -Dremote.snapshot.repository=NOPE -Dremote.release.repository=https://scala-ci.typesafe.com/artifactory/scala-integration/ -Drepository.credentials.id=private-repo -Dscalac.args.optimise=-optimise nightly publish
Buildfile: /home/jenkins/workspace/scala-2.11.x-integrate-bootstrap/build.xml
```

File: news/_posts/2017-04-18-releases-1Q17.md (hunk 1 above)
Comment (adriaanm) [inline news/_posts/2017-04-18-releases-1Q17.md]: Sure, added the link.

**Comment by ashawley** [comment]
> We're still using ant to build the release version of 2.11.

Ahh, ok.  GTK!  Thanks for the clarification.

**Pull request merged**